### PR TITLE
Fix Up Broken Links

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Fix Broken Links Identified by the link verification check

### CODE REVIEW

**Review by seanmcc-msft — APPROVED** [review]

**Review by heaths — APPROVED** [review]
KV LGTM, but should we maybe take this opportunity to define an aka link so we can update it next time it changes without changing a bunch of source files (older files will be wrong still, so the aka link has a benefit for older files as well)?

**Comment by chidozieononiwu** [comment]
> KV LGTM, but should we maybe take this opportunity to define an aka link so we can update it next time it changes without changing a bunch of source files (older files will be wrong still, so the aka link has a benefit for older files as well)?

What will be a good vanity name `azsdk/style-guide-cloud`

**Comment by weshaggard** [comment]
FYI @sima-zhu looks like this PR is fixing some of the broken links I pointed out to you earlier.

File: eng/ignore-links.txt [hunk 1]
```diff
@@ -4,3 +4,4 @@ https://github.com/Azure/azure-digital-twins/blob/private-preview/Documentation/
 https://github.com/Azure/azure-sdk-for-net-pr/tree/feature/IoT-ADT/sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DTDL/DigitalTwins
 https://github.com/Azure/azure-sdk-for-net-pr/blob/feature/IoT-ADT/sdk/iot/Azure.Iot.DigitalTwins/samples/DigitalTwinServiceClientSample/DTDL/Relationships/HospitalRelationships.json
 https://www.nuget.org/packages/Azure.ResourceManager.DigitalTwins
+https://contoso.azureedge.net/urlsigning/test?expires=2145916800&amp;keyid:key1&amp;signature=iTsrLX9rVAIJkSahBA_j5o9Azf5-j331ohxDR1Gx2js=
```
Comment (weshaggard) [inline eng/ignore-links.txt:7]: Why is this link being added to the ignore list?

Comment (chidozieononiwu) [reply eng/ignore-links.txt:7]: Am not sure what it does but it is not a broken link. It does link to something.

Comment (weshaggard) [reply eng/ignore-links.txt:7]: @sima-zhu can you please investigate to see if there is something wrong with our link checker for this link?

**Pull request merged**